lib/tmtheme.py

```diff
@@ -1,12 +1,21 @@
 """Custom color that looks for colors of format `#RRGGBBAA` as `#AARRGGBB`."""
 from mdpopups.coloraide import Color
-from mdpopups.coloraide.spaces.srgb.css import SRGB
-from mdpopups.coloraide.spaces import _parse
-from mdpopups.coloraide import util
-import copy
+from mdpopups.coloraide.spaces.srgb.css import sRGB
+from mdpopups.coloraide.css import parse, serialize
```

Comment:
This module seems not to be part of mdpopups 4.2.2

Reply:
This entire PR is not compatible with the current mdpopups. A new mdpopups will be released along side ExportHtml. We were working off a prerelease of ColorAide for a long while, so the intent is to update mdpopups to the official release which made some breaking changes.

The official, non-prerelease ColorAide uses `sRGB`, not `SRGB`. Sane casing became more important as we started adding colors like `xyY` and such, so instead of just casing the first as we'd normally, `XyY` which doesn't mean what people would think it means, we started just casing color classes in a way that made sense for the color space, `xyY`. It makes it easier for me, and for users, to remember what the casing is, as it is likely cased in the way it is normally used. `sRGB` is normally cased as `sRGB`.